nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/scroll/RepositoryScroll.java

```diff
@@ -41,6 +44,8 @@
  */
 public class RepositoryScroll implements Scroll {
 
+    public static final String SELECT_DOCUMENTS_IN = "SELECT * FROM Document, Relation WHERE ecm:uuid IN ('%s')";
+
```

Comment:
do we need `Document` and `Relation` ? as there is no joins, i am not sure how this query will be interpreted.
furthermore  can you please give a name field with more context `GET_DOCUMENTS_BY_UUID_QUERY` or something else

Reply:
In other hand we don't given a specific methods on this Repo scroller, just call it from outside using the `init` method ... create ... see `org.nuxeo.ecm.core.scroll.TestRepositoryScroll` for examples